mint.json

```diff
@@ -199,6 +199,7 @@
         "guides/articles",
         "guides/articles/restful-api-testing-with-hoppscotch",
         "guides/articles/improving-your-api-workflow",
+        "guides/articles/widgets",
```

Comment:
Can you change the name of the page here as `uplifting-web-experience-with-hoppscotch-widgets` and in the file directory as `uplifting-web-experience-with-hoppscotch-widgets.mdx`.